pkg/apis/psmdb/v1/perconaservermongodbrestore_types.go

```diff
@@ -16,12 +16,13 @@ import (
 type PerconaServerMongoDBRestoreSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	ClusterName string          `json:"clusterName,omitempty"`
-	Replset     string          `json:"replset,omitempty"`
-	BackupName  string          `json:"backupName,omitempty"`
-	Destination string          `json:"destination,omitempty"`
-	StorageName string          `json:"storageName,omitempty"`
-	PITR        *PITRestoreSpec `json:"pitr,omitempty"`
+	ClusterName  string                            `json:"clusterName,omitempty"`
+	Replset      string                            `json:"replset,omitempty"`
+	BackupName   string                            `json:"backupName,omitempty"`
+	BackupSource *PerconaServerMongoDBBackupStatus `json:"backupSource"`
+	Destination  string                            `json:"destination,omitempty"`
+	StorageName  string                            `json:"storageName,omitempty"`
```

Comment:
please delete Destination and StorageName if they not needed

Reply:
deleted `Destination`